### PR TITLE
update owner page styling

### PR DESCRIPTION
**Pull request by ayumitk**
**Pull Request Summary**

Updated the owner page styling

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Changes**

Screenshot 2024-01-22 at 11 04 39 AM

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Thanks @ayumitk all good here, if the grid is ready, please also add it in this page too.

@bobo-k2 So if I don't claim before the new period the first rewards should expire right? Lets test it:)
Screenshot 2024-01-23 at 16 07 44

**Review by bobo-k2 — APPROVED** [review]

File: src/staking-v3/components/Edit.vue [hunk 1]
```diff
@@ -22,16 +21,24 @@
         <div class="box__bottom">
           <span>{{ $t('stakingV3.editYourInfo.text') }}</span>
         </div>
+        <q-tooltip>
+          <span class="text--tooltip">{{ $t('common.comingSoon') }}</span>
+        </q-tooltip>
       </div>
-      <div class="box--link">
+      <a
+        href="https://docs.astar.network/docs/use/dapp-staking/for-devs/create-promotion/"
```
Comment (impelcrypto) [inline src/staking-v3/components/Edit.vue:29]: Let's import the url from `docsUrl`. You can find it at`src/links`.

**Review by impelcrypto — CHANGES_REQUESTED** [review]

File: src/staking-v3/components/Edit.vue (hunk 1 above)
Comment (ayumitk) [reply src/staking-v3/components/Edit.vue]: @impelcrypto Thank you! Fixed in [8117539](https://github.com/AstarNetwork/astar-apps/pull/1143/commits/8117539ff9503c8fd1782ea1252378031518755f)

**Comment by ayumitk** [comment]
@Kahonnohak 
> if the grid is ready, please also add it in this page too.

-> Let me include this in another PR because I need to edit many files, such as other page files and common layout files.

> and make SBY small (16px)

-> Fixed it

**Review by impelcrypto — APPROVED** [review]

**Pull request merged**